models/model.js

```diff
@@ -6,3 +6,17 @@ exports.fetchCategories = () => {
     return result.rows;
   });
 };
+
+exports.fetchReviewId = (review_id) => {
+  return db
+    .query(`SELECT * FROM reviews WHERE review_id = $1;`, [review_id])
```

Comment:
Good to see you've used a paramterized query, and not a template literal, this helps prevent sql injection :-)